src/domain/common/whiteboard/whiteboard.service.ts

```diff
@@ -217,11 +219,16 @@ export class WhiteboardService {
     );
 
     whiteboard.content = JSON.stringify(newContent);
-
+    whiteboard.updatedDate = new Date(); // Make sure that at least the updatedDate is updated (if the content is the same)
```

Comment:
if the content is the same, the same whiteboard will be returned at `line 206`, without an update date, and no subscription will be triggered.
On the other hand, If you have reached this line, then the `updatedDate` will be updated regardlessly.

Try to set the `updateDate` in the database regardless of any whiteboard content changes.

Reply:
The thing is, just a bit later we are doing JSON.parse and JSON.stringify to the whiteboard content so the whiteboard content is never the same. I've debugged it a bit and that `if` is always `false`, it was typeorm the one skipping the save, after the stringify. 
that's why I saw it working well while testing. 
no worries, I'll restructure a bit the code.